internal/commands/app/create.go

```diff
@@ -22,12 +23,47 @@ type CommandCreate struct {
 func (cmd *CommandCreate) Flags(fs *pflag.FlagSet) {
 	fs.StringVar(&cmd.inputs.Project, flagProject, "", flagProjectUsage)
 	fs.StringVarP(&cmd.inputs.Name, flagName, flagNameShort, "", flagNameUsage)
+	fs.StringVarP(&cmd.inputs.From, flagFrom, flagFromShort, "", flagFromUsage)
```

Comment:
Yeah, since the testing is really around the inputs struct (and is set directly) just missed needing this till I realized it was missing from the command.